migrations/versions/59cf8dc3cfde_.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
This migration does nothing. Delete it.

Reply:
It was needed after the merge with your migration file from #429, because it had two HEADS after it, so I had to run `alembic merge heads` https://alembic.sqlalchemy.org/en/latest/branches.html